sklearn/utils/estimator_checks.py

```diff
@@ -250,7 +251,12 @@ def check_estimator(Estimator):
     name = Estimator.__name__
     check_parameters_default_constructible(name, Estimator)
     for check in _yield_all_checks(name, Estimator):
-        check(name, Estimator)
+        try:
+            check(name, Estimator)
+        except SkipTest as message:
+            # the only SkipTest thrown currently results from not
+            # being able to import pandas.
+            warnings.warn(message, ImportWarning)
```

Comment:
Should I add a SkipTestWarning to the [exceptions.py file](https://github.com/scikit-learn/scikit-learn/blob/60a1356f3144f807c5258fa82e29eade2c46c8ed/sklearn/exceptions.py) that inherits from the base `Warning` class? (Would it be in the scope of this PR?)